utils/trivia.py

```diff
@@ -57,3 +60,42 @@ def count(self) -> int:
     #  messages
     def __len__(self) -> int:
         return self.count
+
+
+# Trivia game object
+# The purpose of this object is to hold trivia game data and
+# maintain game state.
+class Trivia:
+
+    # Object for holding the list of questions
+    qlist: QuestionList
+
+    # The inProgress variable is used to keep track of
+    # whether a trivia game is in progress or not.
+    inProgress: bool
```

Comment:
I was wondering, would the Trivia class not also need an object to keep track of the question currently being asked?

Reply:
It would need a data member that would store an object, string, ID or reference to the question being asked so it could appropriately match answers to the question and move on to the next question.